tests/Resources/Common.robot

```diff
@@ -103,4 +103,20 @@ Verify NPM Version
     ...    against an expected version in a given pod/container
     [Arguments]  ${library}  ${expected_version}  ${pod}  ${namespace}  ${container}=""  ${prefix}=""  ${depth}=0
     ${installed_version} =  Run  oc exec -n ${namespace} ${pod} -c ${container} -- npm list --prefix ${prefix} --depth=${depth} | awk -F@ '/${library}/ { print $2}'
-    Should Be Equal  ${installed_version}  ${expected_version}
\ No newline at end of file
+    Should Be Equal  ${installed_version}  ${expected_version}
+Get The Labels Of Build
```

Comment:
This keyword should be at tests/Resources/Page/OCPDashboard/Builds/Builds.robot
It could be renamed to `Get Build Labels`

Reply:
Done

Reply:
I still see `Get Build Labels` in Comon.robot